f5/bigip/tm/cm/trust.py

```diff
@@ -31,13 +31,17 @@ class Add_To_Trust(UnnamedResourceMixin, ExclusiveAttributesMixin,
 
     def __init__(self, cm):
         super(Add_To_Trust, self).__init__(cm)
+        base_uri = type(self).__name__.replace('_', '-').lower()
+        self._meta_data['uri'] =\
+            self._meta_data['container']._meta_data['uri'] + base_uri + '/'
```

Comment:
Let's use `_set_meta_data_uri` here.   If there's some reason that method isn't working then we need to understand it.